h2o-core/src/main/java/water/api/schemas3/ModelParametersSchemaV3.java

```diff
@@ -97,7 +97,8 @@
           " Negative weights are not allowed. Note: Weights are per-row observation weights and do not increase the" +
           " size of the data frame. This is typically the number of times a row is repeated, but non-integer values are" +
           " supported as well. During training, rows with higher weights matter more, due to the larger loss function" +
-          " pre-factor.")
+          " pre-factor. If you set weight == 0, the returned prediction frame at that row is zero. To get an accurate" +
+          " prediction, remove all rows with weight == 0.")
```

Comment:
zain:

Please do a ./gradlew clean build -x test at the h2o-3 directory.  You will noticed that there will be new generated python and R files that you will need to do git add and git commit.

Reply:
zain:  please change this 

If you set weight == 0, the returned prediction frame at that row is zero.

to

If you set weight = 0 for a row, the returned prediction frame at that row is zero and this is incorrect.